great-idea/index.html

```diff
@@ -14,10 +15,97 @@
   <![endif]-->
 </head>
 
-  <body>
+<body>
+<div class="container">  
+  <header>
+    <nav>
+      <a href="">Services</a>
+      <a href="">Product</a>
+      <a href="">Vision</a>
+      <a href="">Features</a>
+      <a href="">About</a>
+      <a href="">Contact</a>
+    </nav>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </header>
+
+
+  <section class="top-container">
+    <div class="left">
+      <h1>
+        Innovation<br>
+        On<br>
+        Demand<br>
+      </h1>
+
+      <button>
+        Get Started
+      </button>
+    </div>
+    <div class="right">
+      <img src="img/header-img.png" alt="Image of a code snippet.">
+    </div>
+  </section>
+
```

Comment:
```suggestion
  <section class="top-container">
    <div class="left">
      <h1>Innovation<br>On<br>Demand<br></h1>
      <button>Get Started</button>      
    </div>

    <div class="right">
      <img src="img/header-img.png" alt="Image of a code snippet.">
    </div>
  </section>
``` i would space these out so you can see whats going on better

Reply:
also you can condense the code down to use less lines